pkgs/tools/networking/networkmanager/fix-paths.patch

```diff
@@ -121,14 +121,14 @@ index 49314cad2e..6d52624699 100644
        '@OUTPUT@',
        nm_property_infos_xml['nmcli'],
 diff --git a/src/tests/client/meson.build b/src/tests/client/meson.build
-index b2e455bbbd..a12ebf212a 100644
+index 6dc0f2a2c8..0a32977a59 100644
 --- a/src/tests/client/meson.build
 +++ b/src/tests/client/meson.build
 @@ -6,7 +6,6 @@ test(
    args: [
      build_root,
      source_root,
 -    python.path(),
+     '--',
```

Comment:
Looks like the implementation is pretty ugly but it should work.

https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/commit/ec454f924c562661b94afaceee7499e734e6314e#089681658d6ce4d7dd3e1d8f800779fb1c537cf5_12_41